pkg/util/nativeutils/eventbus/publisher.go

```diff
@@ -17,18 +16,22 @@ type Publisher interface {
 // Publishing is a fire and forget. If there is no listener for a topic, the
 // messages are lost
 func (bus *EventBus) Publish(topic topics.Topic, m message.Message) {
-	logEB.WithFields(logrus.Fields{
-		"topic":    topic,
-		"category": m.Category(),
-	}).Traceln("publishing on the eventbus")
+	//logEB.WithFields(logrus.Fields{
```

Comment:
removed the publishing on the eventbus log, since its really just all the time logged and the value of it is questionable, suggestions ?

Reply:
Fine to remove